File: Count.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>

int main(){
    char A[20];
    char Q[20];
    int Count = 0;
    int Sum = 0;
    double Avg = 0;
    int Big = -2147483647;
    int Small = 2147483647;

    printf("Type Done To Show Results\n");
    while (2 > 1)
    {
        printf("Enter a Number\n");
        scanf("%s", &A);
        if (strcmp(strlwr(A), "done") == 0)
        {
            break;
        }
        Count += 1;
        printf("%d - %s\n", Count, A);
        int A2 = atoi(A);
        Sum += A2;
        Avg = Sum / Count;
        if (Big < A2)
        {
            Big = A2;
        }
        if (Small > A2)
        {
            Small = A2;
        }
    }
    printf("The Count is - %d\n", Count);
    printf("The Sum is - %d\n", Sum);
    printf("The Average is - %lf\n", Avg);
    printf("The Biggest Number is - %d\n", Big);
    printf("The Smallest Number is - %d\n", Small);
    
    printf("\nType Anything To Quit\n");
    scanf("%s", &Q);

    return 0;
}

File: count.py
Count = 0
Sum = 0
Avg = 0.0
Big = None
Small = None

print("Type Done To Show Results")
while True:
    print("Enter a Number")
    a = input()
    if a.lower() == "done":
        break
    Count += 1
    print(Count, "-", a)
    a2 = int(a)
    Sum += a2
    Avg = Sum / Count
    if Big is None or Big < a2:
        Big = a2
    if Small is None or Small > a2:
        Small = a2

print("The Count is -", Count)
print("The Sum is -", Sum)
print("The Average is -", Avg)
print("The Biggest Number is -", Big)
print("The Smallest Number is -", Small)

print("\nType Anything To Quit")
input()